likeminds-chat-reactnative-integration/ChatSX/components/ExploreFeedItem/index.tsx

```diff
@@ -178,13 +178,15 @@ const ExploreFeedItem: React.FC<Props> = ({
                     chatroomTitle?.color && {
                       color: chatroomTitle?.color,
                     },
-                    chatroomTitle?.fontSize && {
-                      fontSize: chatroomTitle?.fontSize,
-                    },
+                    chatroomTitle?.fontSize
+                      ? {
+                          fontSize: chatroomTitle?.fontSize,
+                        }
+                      : null,
                     chatroomTitle?.fontFamily && {
                       fontFamily: chatroomTitle?.fontFamily,
                     },
-                  ] as TextStyle
+                  ] as TextStyle[]
```

Comment:
Why this change is required?

Reply:
As styles are multiple object bind together. In react native we pass it as an array. If it is a single object we can pass it directly.

It should like this only from the start, but since types got sorted. It raised it now.